pkg/workload/tpcc/tpcc.go

```diff
@@ -753,6 +818,17 @@ func (w *tpcc) Ops(
 	cfg.MaxConnsPerPool = w.connFlags.Concurrency
 	fmt.Printf("Initializing %d connections...\n", w.numConns)
 
+	// Set up the query tracer.
+	var tracer *fileLoggerQueryTracer
+	if w.queryTraceFile != `` {
+		var err error
+		tracer, err = newFileLoggerQueryTracer(w.queryTraceFile)
+		if err != nil {
+			panic(err)
```

Comment:
This function returns an error, we could return the error instead of panicking.

Reply:
Oh good point, updated